platform/features/listview/bundle.js

```diff
@@ -0,0 +1,40 @@
+define([
+    'openmct',
+    './src/controllers/ListViewController',
+    './src/directives/MCTGesture'
+], function (
+    openmct,
+    ListViewController,
+    MCTGesture
+) {
+    openmct.legacyRegistry.register("listview", {
+        "name": "List View Plugin",
+        "description": "Allows folder contents to be shown in list format",
+        "extensions":
+        {
+          "views": [
+            {
+              "key": "list",
+              "type": "folder",
+              "name": "List",
+              "cssClass": "icon-check",
+              "templateUrl": "templates/listview.html"
```

Comment:
Use require to load this template and pass as `template` instead of `templateUrl`.